sys/net/link_layer/ieee802154/security.c

```diff
@@ -0,0 +1,521 @@
+/*
+ * Copyright (C) 2020 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author Fabian Hüßler <fabian.huessler@ovgu.de>
+ * @}
+ */
+
+#include <stdbool.h>
+#include <string.h>
+
+#include "crypto/ciphers.h"
+#include "crypto/modes/ecb.h"
+#include "crypto/modes/cbc.h"
+#include "net/ieee802154_security.h"
+
+const ieee802154_radio_cipher_ops_t ieee802154_radio_cipher_ops = {
+    .set_key = ieee802154_sec_set_key,
+    .ecb = ieee802154_sec_ecb,
+    .cbc = ieee802154_sec_cbc
+};
+
+static inline uint8_t _min(uint8_t a, uint8_t b)
```

Comment:
```suggestion
static inline uint16_t _min(uint16_t a, uint16_t b)
```

quick remedy for encryption with 802.15.4g frames getting stuck.
With this, sending and receiving of large (>255 byte) frames works fine.